### DISCUSSION TITLE
How to detect Qdialog is already open or not?

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#16793] [General - Components / Directives / etc] [answered]
Hi. I use seamless qdialog.. Since it's seamless I can interact with other side of page and when I click open dialog button , it opens new dialog successively. How to prevent this? My code is as follows: 

 ```
function newDiscussion () {
    q.dialog({
        component: DiscussionForm,
        componentProps: {
            formTitle: 'Create Discussion',
            projectId: ''
        }
    }).onOk(() => {
        // router.reload({ only: ['iterations'] })
    }).onCancel(() => {
    // console.log('Cancel')
    }).onDismiss(() => {
    // console.log('I am triggered on both OK and Cancel')
    })
}
```
and my custom dialog component (it's seamless): 

<q-dialog ref="dialogRef" @hide="onDialogHide" seamless position="bottom">
.
.
...
...```

When everyclick to create discussion button it creates new dialog onto existing dialog.

**Comment by ivanjaros** [comment, marked as answer]
Use v-model, duh :D
